tests/microbenchmarks.py

```diff
@@ -266,6 +277,7 @@ def C(self):
             obj.C()
     """), timed=True)
 
+
 def main():
     test_case = TranspileTestCase()
```

Comment:
At column 17: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: utils